robMsg/json/de.json

```diff
@@ -134,6 +134,17 @@
 	"LOOP_FOREVER": "Wiederhole unendlich oft",
 	"MENU_ABOUT": "Über das Open Roberta Lab",
 	"MENU_ABOUT_PROJECT": "Über das Open Roberta Projekt",
+	"MENU_SHORTCUT_IMPORT": "Ctrl + I: Programm importieren",
+	"MENU_SHORTCUT_EXPORT": "Ctrl + E: Programm exportieren",
+	"MENU_SHORTCUT_SAVE": "Ctrl + S: Speichern",
+	"MENU_SHORTCUT_SAVEAS": "Ctrl + Shift + S: Speichern als",
+	"MENU_SHORTCUT_RUN": "Ctrl + R: Auf Roboter laufen",
+	"MENU_SHORTCUT_SOURCE_CODE": "Ctrl + Z: Zeige Quellcode",
+	"MENU_SHORTCUT_EDITOR": "Ctrl + Shift + Z: Quellcode-Editor",
+	"MENU_SHORTCUT_MULTISIM": "Ctrl + Shift + G: Mehrfachrobotersimulation",
+	"MENU_SHORTCUT_OPEN_SIM": "Ctrl + G: Simulationsansicht",
```

Comment:
Please use here "Öffne/schließe die Simulation" as it is defined in the German translation of the MENU_START_SIM key